### ISSUE TITLE
Docker hub endpoint

### ISSUE THREAD

**Issue by grafuls** [niccokunzmann/dockerhub-build-status-image#14]
I am not able to make use of this functionality given that my latest repo is not under the v2 endpoint like here:
 https://github.com/niccokunzmann/dockerhub-build-status-image/blob/27174070c969daad40cbdb66456afb6313583183/python-server/dockerhub_status_image_api.py#L15 My repo buildhistory lives here instead:
https://hub.docker.com/r/grafuls/firefox-java/buildhistory/
Would nice to see some logic there to find the right endpoint.

**Comment by niccokunzmann** [comment]
I also experienced this with my repo, I think. I added it recently and
it does not work.

**Comment by ukos-git** [comment]
also experiencing issues with new repositories:

While [old](https://dockerbuildbadges.quelltext.eu/status.svg?organization=ukos&repository=iputils) repositories work, [newly created](https://dockerbuildbadges.quelltext.eu/status.svg?organization=ukos&repository=sphinx) repositories give an error message:

```python
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/usr/local/lib/python3.5/site-packages/bottle.py", line 1732, in wrapper
    rv = callback(*a, **ka)
  File "dockerhub_status_image_api.py", line 105, in get_svg
    status = get_status(organization, repository, tag)
  File "dockerhub_status_image_api.py", line 80, in get_status
    url = data["next"] # TODO: error handling
KeyError: 'next'
```

**Comment by niccokunzmann** [comment]
Hi, thanks for reporting to you all and for starting the dialog.
I made this repository because it was a use-case for me. Currently, it
is not in my time and interest to improve it myself.
I made this open-source so that in this case, people who have time and
care about this service can improve it themselves (instead of re-writing
a new service from scratch).
As I invite you to use the service, I invite you to share your path
towards fixing this error with the others interested.

In short: Fixing this error will not be done by me. I will deploy a fix
as soon as it is shared with me.

**Comment by sneumann** [comment]
Hi, for the shield.io stuff the solution is in this comment:
https://github.com/badges/shields/issues/1400#issuecomment-469507416
Yours, Steffen
